editor/editor_node.cpp

```diff
@@ -4654,6 +4654,8 @@ void EditorNode::_bind_methods() {
 	ClassDB::bind_method("edit_node", &EditorNode::edit_node);
 	ClassDB::bind_method("_unhandled_input", &EditorNode::_unhandled_input);
 
+	ClassDB::bind_method(D_METHOD("push_item", "object", "property", "inspector_only"), &EditorNode::push_item, DEFVAL(""), DEFVAL(false));
```

Comment:
As stated in [documentation](https://docs.godotengine.org/en/latest/development/cpp/object_class.html#registering-an-object),

> Default values for arguments can be passed in reverse order:
```gdscript
ClassDB::register_method(D_METHOD("methodname", "arg1name", "arg2name"), &MyCustomType::method, DEFVAL(-1)); // default value for arg2name
```

It seems, however, that you passed them in natural order (`DEFVAL("")` for `property`, `DEFVAL(false)` for `inspector_only`).

Reply:
Not sure if this is matter and affect performance. There is lot of places in code where my approach is used..

Reply:
Ok, I will fix it tomorrow

Reply:
It will emit "core\undo_redo.cpp:287 - Error calling method from signal 'push_item': 'EditorNode::push_item': Method expected 2 arguments, but called with 1." if I use your suggestion.

Reply:
Or should I use DEFVAL(-1) twise ?

Reply:
The documentation is unclear, the default arguments are passed from left to right (natural order), but the affected parameters are in reverse order. What I mean is:
```
ClassDB::register_method(D_METHOD("methodname", "arg1name", "arg2name", "arg3name"), &MyCustomType::method, DEFVAL(-1), DEFVAL(""));
```
`arg2name` has default value -1, `arg3name` has default value "", `arg1name` has no default value (hence the "reverse order": two defvals were given, so they match the last two parameters, not the first two).

Reply:
Well, so the documentation is really unclear. I think that instead of "reversed" it should say "right-shifted" or something like this.